be/src/column/fixed_length_column_base.cpp

```diff
@@ -290,7 +290,7 @@ template <typename T>
 std::string FixedLengthColumnBase<T>::debug_string() const {
     std::stringstream ss;
     ss << "[";
-    size_t size = this->size();
+    int size = this->size();
```

Comment:
I think you should change the for loop int to size_t. not change int to size_t in this line.

Reply:
The problem is `size - 1` may overflow if size is type `size_t`, changing `int i` to `size_t i` don't work out

Reply:
O，I see